presets/shared.yml

```diff
@@ -223,6 +223,8 @@ restProtocol: HTTP
 restSSLPath: '/symbol-workdir'
 restSSLKeyFileName: 'restSSL.key'
 restSSLCertificateFileName: 'restSSL.crt'
+restNodeMetadata:
+  _info: "Node metadata"
```

Comment:
Do you know what's the default use case for this endpoint?
It would be nice to autofill the default values.

Reply:
For example, at this moment, additional information about the node listed in https://symbol-tools.com/symbolTools/view/tool/nodeList.html are added manually via an additional configuration file (https://symbol-tools.com/nodeSetting.json generated here https://symbol-tools.com/symbolTools/view/tool/nodeAdminManual.html). This is possible if you host the node, but Bootstrap will now support it. 
I don't know what default keys/values should be set. Default values may be wrongly read as set by the operator since it's  :thinking: